authorization-service/.env

```diff
@@ -0,0 +1 @@
+anderysenuta=TEST_PASSWORD
```

Comment:
It would be better not to commit **.env** files

Reply:
yep.. thank you